doxygen/examples/stdin_handler.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Copyright (C) 2015 Simon Newton
+ */
+//! [Example] NOLINT(whitespace/comments)
+#include <ola/base/Init.h>
+#include <ola/io/SelectServer.h>
+#include <ola/io/StdinHandler.h>
+#include <iostream>
+
+using ola::io::SelectServer;
+using std::cout;
+using std::endl;
+
+class ExampleStdinHandler {
+ public:
+  ExampleStdinHandler();
+  ~ExampleStdinHandler();
+
+  void Run() { m_ss.Run(); }
+
+ private:
+  ola::io::SelectServer m_ss;
+  ola::io::StdinHandler m_stdin_handler;
+
+  void Input(int c);
+};
+
+ExampleStdinHandler::ExampleStdinHandler()
+  : m_stdin_handler(&m_ss,
```

Comment:
Put it in the class section rather than a separate method:

class ExampleStdinHandler {
public:
   ExampleStdinHandler() 
      :  m_stdin_handler(&m_ss,  ...) {
 }